index.html

```diff
@@ -3,31 +3,33 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link href="https://fonts.googleapis.com/css?family=Open%20Sans" rel='stylesheet'>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&display=swap" rel="stylesheet">
```

Comment:
I'm not totally sure if we're using the right font here. Let's revisit it once we get the responsiveness figured out.